patches/chrome-browser-resources-bookmarks-toolbar.js.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/bookmarks/toolbar.js b/chrome/browser/resources/bookmarks/toolbar.js
+index 4a2457293c6add39ba89a6f8e0979a674d41e792..c9bcbe9f776fa171e156d717411c27f9dcb2c4f0 100644
+--- a/chrome/browser/resources/bookmarks/toolbar.js
++++ b/chrome/browser/resources/bookmarks/toolbar.js
+@@ -3,7 +3,7 @@
+ // found in the LICENSE file.
+ 
+ import 'chrome://resources/cr_elements/cr_icon_button/cr_icon_button.m.js';
+-import 'chrome://resources/cr_elements/cr_toolbar/cr_toolbar.m.js';
```

Comment:
I think we want a follow-up here to replace cr_toolbar.m.js instead

Reply:
these can be done in a follow-up since they didn't increase the number of patches